.ci.yaml

```diff
@@ -417,12 +476,16 @@ targets:
         {
           "sdk_version": "15a240d"
         }
+    # Do not remove(https://github.com/flutter/flutter/issues/144644)
+    # Scheduler will fail to get the platform
     drone_dimensions:
       - os=Mac-13|Mac-14
 
+  # Avoid using a Mac orchestrator to save ~5 minutes of Mac host time.
   - name: Linux mac_clangd
```

Comment:
Should this have cores: "8"?

Reply:
This one doesn't have `release_build: true`, so it won't run on dash-internal. Adding `cores: "8"` would be harmless here, but isn't necessary.

Reply:
Happy to add it everywhere or add more comments? Not sure what would be the most useful.

Reply:
ack, this is good as is